digits/extensions/view/__init__.py

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) 2016, NVIDIA CORPORATION.  All rights reserved.
+from __future__ import absolute_import
+
+from . import rawData
+
+view_extensions = [
+    # set show=True if extension should be listed in known extensions
+    {'class': rawData.Visualization, 'show': True},
```

Comment:
That's the rawData visualization - it's just showing the raw data. To see the little arrows you need to select the gradient visualization (which isn't in this PR - it's in https://github.com/NVIDIA/DIGITS/pull/735/commits/e8001f863c93ddceae6875362182399e60f8af75 if you're curious to try). The screenshot was on the PR description for #735 - sorry it's confusing!)